apps/server-web/src/renderer/pages/setup/AdvancedSetting.tsx

```diff
@@ -0,0 +1,151 @@
+import { useTranslation } from 'react-i18next';
+import { IServerSetting } from '../../libs/interfaces';
+import { useState } from 'react';
+import { SettingPageTypeMessage } from '../../libs/constant';
+type Props = {
+  back: () => void;
+};
+
+const AdvancedSetting = (props: Props) => {
+  const { t } = useTranslation();
+  const [serverSetting, setServerSetting] = useState<IServerSetting>({
+    PORT: 3000,
+    GAUZY_API_SERVER_URL: 'http://localhost:3030',
+    NEXT_PUBLIC_GAUZY_API_SERVER_URL: 'http://localhost:3030',
+  });
+  const saveSetting = (e: any) => {
+    e.preventDefault();
+
+    window.electron.ipcRenderer.sendMessage('setting-page', {
+      data: serverSetting,
+      type: SettingPageTypeMessage.saveSetting,
+      isSetup: true,
+    });
+  };
+  const handleChange = (event: any) => {
+    const { id, value } = event.target;
+    setServerSetting((prevData: any) => ({ ...prevData, [id]: value }));
+  };
+  return (
+    <>
+      <div className="text-center mt-8 mb-12 text-gray-700 dark:text-gray-200">
+        <h1 className="text-2xl font-bold mb-2">
+          Are you ready for some advanced settings?
+        </h1>
+        <p className="text-gray-500 dark:text-gray-400">
+          This is the final step where you can get to the nitty-gritty of your
+          Gauzy Platform.
```

Comment:
@syns2191 please in next PRs make sure no mention of "Gauzy Platform" directly (i.e. hardcoded), but instead you use translation files!

Reply:
noted. i will update on next PR